scripts/install.sh

```diff
@@ -9,7 +9,7 @@ if ! [[ -x "$(command -v danger)" ]]; then
 	fi
 
 	echo "Installing danger"
-	npm install -g danger
+	sudo npm install -g danger
```

Comment:
you should check if sudo is enabled before using it

```bash
if [[ -n "$sudo" ]]; then
  sudo npm install -g danger
else
  npm install -g danger
fi
```

Reply:
we do not need to to the check.. before we are doing `sudo -v && sudo="true" || sudo=""`
that means if sudo is not present is aliased as `""`

Reply:
woops! ok got it! ignore this, i'll fix it